bundles/org.openhab.binding.mqtt.awtrixlight/README.md

```diff
@@ -303,9 +309,9 @@ Switch Display_Sound "Sound" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myA
 Switch Display_AutoBrightness "Auto Brightness" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:autoBrightness" }
 Number:Temperature Display_Temperature "Temperature [%.1f °C]" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:temperature" }
 Number:Dimensionless Display_Humidity "Humidity [%d %%]" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:humidity" }
-Number Display_Battery "Battery Level [%d %%]" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:batteryLevel" }
+Number Display_Battery "Battery Level [%d %%]" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:battery-level" }
 Switch Display_LowBattery "Low Battery" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:lowBattery" }
-Number:Dimensionless Display_WiFi "WiFi Signal [%d %%]" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:rssi" }
+Number Display_WiFi "WiFi Signal [%d dBm]" (gAwtrix) { channel="mqtt:awtrix-clock:myBroker:myAwtrix:rssi" }
```

Comment:
Displaying this as a percentage is incorrect. Value comes in with values like `-49`, `-52`.
The existing example makes it look like -4900%, -5200%

Reply:
Ah. That would explain the error
> Failed to update item 'AwtrixBedroomWiFi' because '-53 dBm' could not be converted to the item unit 'one'
when I had it dimensionless?

Can confirm changing to `Number:Power` works. Is that all you meant?
I didn't realise adding `dBm` in the formatter was actually assigning a dimension unit

Reply:
Thanks - applied `:Power` and `,unit="dBm"`
When applied to my own config - indeed the unit's changed
> 2025-08-10 20:48:10.925 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'AwtrixBedroomWiFi' changed from 3.981071705534966E-9 W to -48 dBm